test/streamqflite_test.dart

```diff
@@ -7,17 +7,17 @@ import 'package:streamqflite/streamqflite.dart';
 import "package:flutter_test/flutter_test.dart";
 
 void main() {
-  Database db;
-  StreamDatabase streamDb;
+  Database? db;
+  StreamDatabase? streamDb;
```

Comment:
I think you can declare these as late to avoid the unwrapping latter https://dart.dev/null-safety/understanding-null-safety#late-variables